front/app/component-library/utils/styleUtils.ts

```diff
@@ -207,6 +207,22 @@ export const stylingConsts = {
   border: `1px solid ${colors.divider}`,
 };
 
+type DefaultInputStyleProps = {
+  theme: MainThemeProps;
+  height?: string | number;
+  size?: InputSize;
+};
+
+const getHeight = (props: DefaultInputStyleProps) => {
```

Comment:
Allowing for a custom height. Useful when the input is used as part of another input